pilot/pkg/model/context.go

```diff
@@ -510,6 +510,11 @@ const (
 	// Alpha in 1.1, based on feedback may be turned into an API or change. Set to "1" to enable.
 	NodeMetadataHTTP10 = "HTTP10"
 
+	// NodeMetadataFallthroughRoute provides an option to add a final wildcard match for routes.
+	// When ALLOW_ANY traffic policy is used, a Passthrough cluster is used.
+	// When REGISTRY_ONLY traffic policy is used, a 502 error is returned.
+	NodeMetadataFallthroughRoute = "sidecar.istio.io/enableFallthroughRoute"
```

Comment:
We dont need this because the existing one was supposed to handle it.. 
At the very least, node metadata is an environment variable.. So please use ENABLE_ALLOW_ANY_HTTP or something to that effect

Reply:
Come to think of it, I dont think we need node metadata. That seems out of place with the fact that outbound traffic policy does not have node metadata. Lets remove this and just have the pilot level flag.

Reply:
The intent was to allow the flag to be configured on a per service level, and would be removed once we consider this stable. We have a similar pattern with htrp 1.0 support

Reply:
Does this need to be an option at all? How about just turn it on, by default. The current implementation of `ALLOW_ANY` is arguably a bug, since it really does not allow any. And, since it's just becoming less restrictive than current implementation, it's not a breaking change. Right?

Reply:
I agree for the most part. The reason behind the flag is this is a pretty fairly large change (in terms of behavior, not code) being released late into a release cycle (1.1.3?). The change has not been tested to the degree that the rest of 1.1 has been - one potential issue is that we are pushing more config to each proxy, and we have not measured the performance impact of this.

Maybe we could then this on by default and have a flag to disable it, but I would err on the side of caution unless others disagree.

A potential compromise could be to have the flag disabled by default for one release and in 1.1.4 (we are aiming for weekly releases I believe), once we've done more testing, we can enable it by default.

@mandarjog any thoughts since this relates to performance?

Reply:
This is per proxy flag.. which I think is more risky. WE dont have per proxy flag for OutboundtrafficPolicy (top level thing). So providing a per proxy flag for some internal function of outboundTrafficPolicy will cause more confusion as users will tend to think that this "should" work, when in reality it is gated by the mesh level outbound traffic policy. And I am not so keen on adding yet another sidecar variable for outbound traffic policy. @robertpanzer has a nice PR that enables this though sidecar API instead. 

At pilot level, you already have a flag which as @frankbu says, should be enabled by default as we have explicitly advertised that mesh's behavior is allow by default [without yoru fix here, that behavior will break].